nestkernel/source_table.cpp

```diff
@@ -320,67 +364,59 @@ nest::SourceTable::get_next_target_data( const thread tid,
     current_source.set_processed( true );
 
     // we need to set a marker stating whether the entry following this
-    // entry, if existent, has the same source; start by assuming it
-    // has a different source, only change if necessary
-    kernel().connection_manager.set_has_source_subsequent_targets(
-      current_position.tid, current_position.syn_id, current_position.lcid, false );
-    if ( ( current_position.lcid + 1
-             < static_cast< long >( sources_[ current_position.tid ][ current_position.syn_id ].size() )
-           and sources_[ current_position.tid ][ current_position.syn_id ][ current_position.lcid + 1 ].get_node_id()
-             == current_source.get_node_id() ) )
+    // entry, if existent, has the same source
+    kernel().connection_manager.set_has_source_subsequent_targets( current_position.tid,
```

Comment:
Shouldn't the setter be called `set_source_has_subsequent_targets` to be good English? And maybe `set_source_has_more targets` for legibility?

Reply:
i've just added the setter prefix `set_` to the name of the getter `has_source_...` (https://github.com/nest/nest-simulator/blob/master/nestkernel/connection.h#L269) causing this terrible function name. i thought in terms of consistency between getter and setter this would justify the bad language, but i'd be willing to be overruled.

Reply:
I think `source_has_more_targets` would be a much better getter name as well.